### PR TITLE
Case updates for extension parsing.

### PR DESCRIPTION
**Pull request by toddholmberg**
I added two cases to solve some extension parsing errors I ran across. There are two additional DER files in the test/data/der directory for testing these errors.

The errors and solutions are described in issues #8 and #9.

### CODE REVIEW

File: test/easy_ssl_test.exs [hunk 1]
```diff
@@ -23,7 +23,7 @@ defmodule EasySSLTest do
       |> Enum.each(fn cert_filename ->
             original_cert = File.read!(@der_cert_dir <> cert_filename)
               |> EasySSL.parse_der
-
+            IO.inspect original_cert
```
Comment (Fitblip) [inline test/easy_ssl_test.exs:26]: Just FYI I'm going to merge this but remove this later before I release the next version.

**Comment by Fitblip** [comment]
Nice, this looks great. Huge thanks for implementing that and opening a PR! I'm going to merge this along with some other changes and I'll let you know when I finish cutting a new version 👍

**Comment by Fitblip** [comment]
Should be live! https://hex.pm/packages/easy_ssl/1.2.1

Thanks again

File: test/easy_ssl_test.exs (hunk 1 above)
Comment (toddholmberg) [reply test/easy_ssl_test.exs:26]: Sorry about that! Thanks for merging this.

**Pull request merged**